Assets/VRTK/Scripts/VRTK_MoveInPlace.cs

```diff
@@ -0,0 +1,387 @@
+
+        // Keeps track of the play area's last Y position.  This number is used to determine the delta position of the play area between updates.
+        // the delta y-position is then subtracted by all calculations to discount the movement of the play area from the trackable's movements. 
+        private float playAreaPreviousY = 0f;
+
+        // Used to determine the direction when using a decoupling method.
+        private Vector3 initalGaze = Vector3.zero;
+
+        // The current move speed of the player.  If Move In Place is not active, it will be set to 0.00f.
+        private float curSpeed = 0.00f;
+
+        // The current direction the player is moving.  If Move In Place is not active, it will be set to Vector.zero.
+        private Vector3 direction = new Vector3();
+
+        // True if Move In Place is currently engaged.
+        private bool active = false;
+
+        /// <summary>
+        /// Set your control options and modify the trackables to match.
+        /// </summary>
+        /// <param name="givenControlOptions">The control option you wish to set the current control options to.</param>
+        public void SetControlOptions(ControlOptions givenControlOptions)
+        {
+            controlOptions = givenControlOptions;
+            trackedObjects.Clear();
+
+            if (controlOptions.Equals(ControlOptions.HeadsetAndControllers) || controlOptions.Equals(ControlOptions.ControllersOnly))
+            {
+                trackedObjects.Add(controllerLeftHand.transform);
+                trackedObjects.Add(controllerRightHand.transform);
+            }
+
+            if (controlOptions.Equals(ControlOptions.HeadsetAndControllers) || controlOptions.Equals(ControlOptions.HeadsetOnly))
+            {
+                trackedObjects.Add(headset.transform);
+            }
+        }
+
+        /// <summary>
+        /// The GetMovementDirection method will return the direction the player is moving.
+        /// </summary>
+        /// <returns>Returns a vector representing the player's current movement direction.</returns>
+        public Vector3 GetMovementDirection()
+        {
+            return direction;
+        }
+
+        /// <summary>
+        /// The GetSpeed method will return the current speed the player is moving at.
+        /// </summary>
+        /// <returns>Returns a float representing the player's current movement speed.</returns>
+        public float GetSpeed()
+        {
+            return curSpeed;
+        }
+
+        private void Awake()
+        {
+            controllerLeftHand = VRTK_DeviceFinder.GetControllerLeftHand();
+            controllerRightHand = VRTK_DeviceFinder.GetControllerRightHand();
+            headset = VRTK_DeviceFinder.HeadsetTransform();
+
+            SetControlOptions(controlOptions);
+        }
+
+        private void OnEnable()
+        {
+            touchpadPressed += DoTouchpadDown;
```

Comment:
Ok, probably makes sense that you have to activate it, otherwise any movement of the controllers would have you skating around everywhere! :)